data/types/calculator/calculator_test.go

```diff
@@ -1622,6 +1625,13 @@ var _ = Describe("Calculator", func() {
 					pointer.FromString("mg/dl"),
 					func(datum *calculator.Calculator, units *string) { datum.BolusID = nil },
 				),
+				Entry("Invalid Carb Units",
+					pointer.FromString("mg/dl"),
```

Comment:
```suggestion
```
Line not required

Reply:
This is confusing - but I believe what I have is correct.  This is just creating a unit parameter.  This is actually not used in this test case - but necessary.  We are setting the `CarbUnit` parameter to invalid in the func - so the test should work fine.